web/template/add-assurance-visit.gotmpl

```diff
@@ -1,10 +1,14 @@
 {{ template "page" . }}
+
+{{ define "page-name" }}Add assurance visit{{ end }}
```

Comment:
What is the benefit of this change? And if it is required, can it be passed in as a variable rather than defined at the start of every template?